__docs__/wonder-blocks-button/button.stories.tsx

```diff
@@ -27,26 +26,6 @@ import ComponentInfo from "../../.storybook/components/component-info";
 import ButtonArgTypes from "./button.argtypes";
 import {ThemeSwitcherContext} from "@khanacademy/wonder-blocks-theming";
 
-/**
```

Comment:
**note:** Got rid of this b/c it's duplicated. The source of truth is the code in `button.tsx`.